webpack.config.js

```diff
@@ -132,6 +134,8 @@ module.exports = function(opts) {
   config.module = {
     rules: [
       babelLoader,
+      awesomeTypescriptLoader,
+      sourceMapLoader,
```

Comment:
where are these 2 loaders defined?

Reply:
They should of been removed I'll push it up again